README.md

```diff
@@ -1,58 +1,73 @@
+
+## Project Vision
 It's the vision of this project to provide a comprehensive, performant, and trustworthy platform to transform the way that the CMS enterprise shares and uses data.
-This aligns with the overarching mission to enable the CMS Enterprise to drive innovation in data sharing so that beneficiaries and their healthcare partners have the data they
-need to make informed decisions about their healthcare.
 
+## Agency Mission
+This aligns with the overarching mission to enable the CMS Enterprise to drive innovation in data sharing so that beneficiaries and their healthcare partners have the data they need to make informed decisions about their healthcare.
+
+## Repo Structure
 The high-level purpose and location of each piece of the project is listed below.
 
 * [apps](apps) - contains the source code for each of the deployed BFD applications
-  * [bfd-data-fda](apps/bfd-data-fda) - downloads FDA Drug code names into a resource used during BFD drug code name lookups
-  * [bfd-data-npi](apps/bfd-data-npi) - downloads CMS NPI (National Provider Identification) names into a resource used during BFD NPI lookups
-  * [bfd-db-migrator](apps/bfd-db-migrator) - application for safely applying schema updates and data migrations to the BFD database
-  * [bfd-model](apps/bfd-model) - contains data models used throughout the BFD project
-  * [bfd-pipeline](apps/bfd-pipeline) - application for loading claim data from the provider into the BFD database
-  * [bfd-server](apps/bfd-server) - application for serving the BFD database data in FHIR format to users via the BFD API
-  * [bfd-shared-test-utils](apps/bfd-shared-test-utils) - utilities shared across BFD projects used in testing
-  * [bfd-shared-utils](apps/bfd-shared-utils) - utilities shared across BFD projects used in the application code
-  * [utils](apps/utils) - non-application scripts used for testing, development, and database management
+ * [bfd-data-fda](apps/bfd-data-fda) - downloads FDA Drug code names into a resource used during BFD drug code name lookups
+ * [bfd-data-npi](apps/bfd-data-npi) - downloads CMS NPI (National Provider Identification) names into a resource used during BFD NPI lookups
+ * [bfd-db-migrator](apps/bfd-db-migrator) - application for safely applying schema updates and data migrations to the BFD database
+ * [bfd-model](apps/bfd-model) - contains data models used throughout the BFD project
+ * [bfd-pipeline](apps/bfd-pipeline) - application for loading claim data from the provider into the BFD database
+ * [bfd-server](apps/bfd-server) - application for serving the BFD database data in FHIR format to users via the BFD API
+ * [bfd-shared-test-utils](apps/bfd-shared-test-utils) - utilities shared across BFD projects used in testing
+ * [bfd-shared-utils](apps/bfd-shared-utils) - utilities shared across BFD projects used in the application code
+ * [utils](apps/utils) - non-application scripts used for testing, development, and database management
```

Comment:
This should probably be reverted, or the indentation increased so that these are sub-bullets of `apps` so that it is obvious these modules are within the `apps` directory.

Reply:
fixed